### PR TITLE
Cecile/desktop cecile fixes 1

### PR DESCRIPTION
**Pull request by cecileboucheron**
@keybase/react-hackers 
My first mobile PR! #phew
You guys should definitely double check my flow hack on `TabBarButtonSource`, not sure if that was the right way to do. Eventually we will remove the Avatar from the main nav bar completely (with Search v3).

### CODE REVIEW

File: shared/common-adapters/tab-bar.js.flow [hunk 1]
```diff
@@ -30,7 +30,7 @@ export type ItemProps = Exact<{
 
 export class TabBarItem extends Component<void, ItemProps, void> { }
 
-export type TabBarButtonSource = {type: 'icon', icon: IconType} | {type: 'avatar', avatar: React$Element<*>} | {type: 'nav', icon: IconType}
+export type TabBarButtonSource = {type: 'icon', icon: IconType} | {type: 'avatar'} | {type: 'nav', icon: IconType}
```
Comment (chrisnojima) [inline shared/common-adapters/tab-bar.js.flow]: really this should be `{type: 'avatar', username: string}` and remove username from below

File: shared/tab-bar/index.render.native.js [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
-  />
-
   return (
     <Box style={stylesTabBar}>
       <TabBarButton
         selected={selectedTab === profileTab}
         onClick={() => onTabClick(profileTab)}
         badgeNumber={badgeNumbers[profileTab]}
-        source={{type: 'avatar', avatar}}
+        source={{type: 'avatar'}}
```
Comment (chrisnojima) [inline shared/tab-bar/index.render.native.js]: then here it should be `type: 'avatar', username}`

**Comment by keybase-ci-visdiff** [comment]
The commits 395c35d26a57...e338b29981d1 introduce visual changes on linux. <details><summary>:mag_right: 4 changed</summary>

 * Changed: **Tab_Bar-Custom_Buttons_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/Tab_Bar-Custom_Buttons_1-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/Tab_Bar-Custom_Buttons_1-e338b29981d1.png)  
   Tab_Bar-Custom_Buttons_1 comparison
 * Changed: **Tab_Bar-Custom_Buttons_0** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/Tab_Bar-Custom_Buttons_0-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/Tab_Bar-Custom_Buttons_0-e338b29981d1.png)  
   Tab_Bar-Custom_Buttons_0 comparison
 * Changed: **App_Tab_Bar-Search_Active** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/App_Tab_Bar-Search_Active-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/App_Tab_Bar-Search_Active-e338b29981d1.png)  
   App_Tab_Bar-Search_Active comparison
 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/App_Tab_Bar-Normal-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492727248982-395c35d26a57-e338b29981d1-linux/App_Tab_Bar-Normal-e338b29981d1.png)  
   App_Tab_Bar-Normal comparison
</summary>

**Comment by cecileboucheron** [comment]
@chrisnojima Fixed desktop, I'll let you double check

**Comment by keybase-ci-visdiff** [comment]
The commits 395c35d26a57...eddb59d2d3ed introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Tab_Bar-Custom_Buttons_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492731909997-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_1-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492731909997-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_1-eddb59d2d3ed.png)  
   Tab_Bar-Custom_Buttons_1 comparison
 * Changed: **Tab_Bar-Custom_Buttons_0** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492731909997-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_0-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492731909997-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_0-eddb59d2d3ed.png)  
   Tab_Bar-Custom_Buttons_0 comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 395c35d26a57...eddb59d2d3ed introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Tab_Bar-Custom_Buttons_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492817235793-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_1-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492817235793-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_1-eddb59d2d3ed.png)  
   Tab_Bar-Custom_Buttons_1 comparison
 * Changed: **Tab_Bar-Custom_Buttons_0** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492817235793-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_0-395c35d26a57.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1492817235793-395c35d26a57-eddb59d2d3ed-linux/Tab_Bar-Custom_Buttons_0-eddb59d2d3ed.png)  
   Tab_Bar-Custom_Buttons_0 comparison
</summary>

**Comment by chrisnojima** [comment]
seems fine, but you have to fix merge conflicts now

**Pull request merged**